news/2024/2024-06-24-4-digit-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,125 @@
+
+> Hello, I'm BlankTap. I'm a bit of a tournament player and recently more known as a commentator, probably one that gets a bit too excited at times. I've been commentating actively for one and half years now and had the chance to contribute regarding 4WC's casting.
+
+**BlankTap** has been making a name for himself in recent years thanks to his high-energy commentary, and he was an integral part of the casting team during this year's 4WC.
+
+---
+
+*What were your thoughts participating in your first world-cup-style tournament as a commentator? How did you prepare yourself?*
+
+> I was very excited and basically in disbelief. The overturning of what's initially a rejection of my commentating application energized me up to prove that this will not be a decision to be regretted upon. I really do want to say thank you and that I am eternally indebted to those who believed in me that made this possible.
+> 
+> Regarding preparation, I don't think I can explain much other than that I basically did nearly nothing. I've only discovered the archetypes of commentating recently and while this wouldn't fly when you're an analyst or a color caster, play by play casting needs much less intricate knowledge of a team's strategy or skillset and rather how the current map will go in which I do play some of the mappool for fun and I do have some knowledge regarding how a map generally would go for various skillset from the experience I've accumulated as a tournament player myself. In short, my preparation consists of filling up my water bottle, sitting comfortably in my chair, and going hard when the moment needs it to be.
+
+*This year's 4WC had a lot of upsets in the earlier rounds. Do you believe that this is something that could happen more frequently from now on and does it make commentating more exciting for you?*
+
+> Regarding the plentiful early round upsets that were witnessed in this iteration, it really has to do with how the earlier rounds consists of lower star rating maps and the big focus on a player's consistency rather than their skillcap. I do think that every country is as good as each other up until a certain star rating. There are no safe rounds in a world cup where every nation sends in their best of the best. High seeding is nothing more but an intimidation factor if you don't back it up.
+> 
+> Nonetheless, teams do get high seedings not without a merit. I don't think upsets are something that will happen more often but only as often as or less than it is now. If upsets do spike up in quantity, it is only because of an erroneous early round pooling that deviates completely from the qualifier (Ex. speed qualifier into full gimmick RO32). Of course, its frequency is not what makes an upset exciting. We all love an underdog story, the David against the Goliath, this is no exception especially when it comes to a tiebreaker. At that point, the audience and the casters have won regardless of the result.
```

Comment:
```suggestion
> Nonetheless, teams do get high seedings not without a merit. I don't think upsets are something that will happen more often but only as often as or less than it is now. If upsets do spike up in quantity, it is only because of an erroneous early round pooling that deviates completely from the qualifier (Ex. speed qualifier into full gimmick RO32). Of course, its frequency is not what makes an upset exciting. We all love an underdog story, the David against the Goliath, and this is no exception especially when it comes to a tiebreaker. At that point, the audience and the casters have won regardless of the result.
```